testez-cli/src/vfs.rs

```diff
@@ -0,0 +1,89 @@
+//! Simple in-memory representation of a filesystem that can be unpacked.
+//!
+//! Some parts of this code are used in the tool's build script mostly, while
+//! other parts are used in the tool itself.
+
+#![allow(unused)]
```

Comment:
Is this something that you need when you only use something in a build.rs?

Reply:
I should document this file a little better!

I have `#![allow(unused)]` here since half the file is used only in `build.rs` and half of it is only used at runtime, but the structs are used in both cases. This is something that should be a crate, but I'm not sure it's polished enough yet.